ear/src/main/resources/s3binaries/nxserver/config/s3directupload-config.xml.nxftl

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0"?>
+
+<component name="s3DirectUpload" version="1.0.0">
+
+<#if "${nuxeo.s3storage.useDirectUpload}" == "true">
+
+  <require>org.nuxeo.ecm.core.storage.cloud.managment.contrib</require>
+
+  <extension target="org.nuxeo.ecm.core.transientstore.TransientStorageComponent" point="store">
+    <store name="s3TransientStore" class="org.nuxeo.ecm.core.transientstore.keyvalueblob.KeyValueBlobTransientStore">
+      <targetMaxSizeMB>1</targetMaxSizeMB>
+      <absoluteMaxSizeMB>-1</absoluteMaxSizeMB>
+      <firstLevelTTL>120</firstLevelTTL>
+      <secondLevelTTL>10</secondLevelTTL>
+    </store>
+  </extension>
+
+  <extension target="org.nuxeo.ecm.core.blob.BlobManager" point="configuration">
+    <blobprovider name="s3TransientStore">
+      <class>org.nuxeo.ecm.core.storage.sql.S3BinaryManager</class>
+      <property name="awsid">${nuxeo.s3storage.direct.awsid}</property>
+      <property name="awssecret">${nuxeo.s3storage.direct.awssecret}</property>
+      <property name="bucket">${nuxeo.s3storage.direct.bucket}</property>
+      <property name="region">${nuxeo.s3storage.direct.region}</property>
+      <property name="bucket.prefix">/</property>
+      <property name="transient">true</property>
```

Comment:
This one is important and not related to S3. Let's separate it from the others and add a comment like `<!-- this blob provider is used for transient storage -->`